README.md

```diff
@@ -64,32 +64,33 @@ The following options can be passed to `PajaritoSolver()` to modify its behavior
   * `init_exp::Bool` **C** Start with several outer approximation cuts on the exponential cones
   * `init_sdp_lin::Bool` **C** Use SDP initial linear cuts
   * `init_sdp_soc::Bool` **C** Use SDP initial SOC cuts (if MIP solver supports MISOCP)
+  * `scale_subp_cuts::Bool` **C** Use scaling for subproblem cuts based on subproblem status
   * `viol_cuts_only::Bool` **C** Only add cuts that are violated by the current MIP solution (may be useful for MSD algorithm where many cuts are added)
-  * `proj_dual_infeas::Bool` **C** Project dual cone infeasible dual vectors onto dual cone boundaries
-  * `proj_dual_feas::Bool` **C** Project dual cone strictly feasible dual vectors onto dual cone boundaries
-  * `prim_cuts_only::Bool` **C** Do not add dual cuts
-  * `prim_cuts_always::Bool` **C** Add primal cuts at each iteration or in each lazy callback
-  * `prim_cuts_assist::Bool` **C** Add primal cuts only when integer solutions are repeating
-  * `tol_zero::Float64` **C** Tolerance for setting small absolute values in duals to zeros
-  * `tol_prim_zero::Float64` **C** Tolerance level for zeros in primal cut adding functions (must be at least 1e-5)
-  * `tol_prim_infeas::Float64` **C** Tolerance level for cone outer infeasibilities for primal cut adding functions (must be at least 1e-5)
-  * `tol_sdp_eigvec::Float64` **C** Tolerance for setting small values in SDP eigenvectors to zeros (for cut sanitation)
-  * `tol_sdp_eigval::Float64` **C** Tolerance for ignoring eigenvectors corresponding to small (positive) eigenvalues
+  * `prim_cuts_only::Bool` **C** Do not add subproblem cuts (if `prim_cuts_always` and `prim_cuts_assist`)
+  * `prim_cuts_always::Bool` **C** Add primal cuts at each iteration or in each lazy callback (if `prim_cuts_assist`)
+  * `prim_cuts_assist::Bool` **C** Add primal cuts only when integer solutions are repeating or when conic solver fails
+  * `tol_zero::Float64` **C** Tolerance for small epsilons as zeros
+  * `tol_prim_infeas::Float64` **C** Absolute tolerance level for cone outer infeasibilities, should be equal to MIP solver absolute linear constraint feasibility tolerance for best performance (does not apply to feasibility of the conic solver solutions)
 
 **Pajarito is not yet numerically robust and may require tuning of parameters to improve convergence.** If the default parameters don't work for you, please let us know.
 
+Note that for the conic algorithm, Pajarito usually returns a solution constructed from one of the conic solver's feasible solutions. Since the conic solver is not subject to the same feasibility tolerances as the MIP solver (which should match the absolute feasibility tolerance `tol_prim_infeas`), Pajarito's solution will not necessarily satisfy `tol_prim_infeas`.
+
+Note that MIP solver integrality tolerance should typically be tightened, for example to 1e-8, for improved Pajarito performance.
```

Comment:
For the integrality tolerance, I believe so, as I don't believe any changes made are obviously relevant to that. Certainly they were relevant to the constraint feas tol. I have not tried changing integrality tolerances from 1e-8.